server/schema/comment/comment.query.js

```diff
@@ -0,0 +1,76 @@
+const {
+  GraphQLObjectType,
+  // GraphQLScalarType,
+  // GraphQLUnionType,
+  // GraphQLInputObjectType,
+  // GraphQLEnumType,
+  // GraphQLInterfaceType,
+  // GraphQLSchema,
+  GraphQLNonNull,
+  // GraphQLError,
+  GraphQLList,
+  GraphQLString,
+  GraphQLID,
+  // GraphQLBoolean,
+  GraphQLInt,
+  // GraphQLFloat,
+  // GraphQLDate,
+  // GraphQLTime,
+  // GraphQLDateTime,
+  // GraphQLJSON,
+  // GraphQLJSONObject,
+} = require('../scalars');
+const { getListOfComments, getCommentById, countOfComments } = require('./comment.resolver');
+
+const CommentType = require('./comment.type');
+
+module.exports = new GraphQLObjectType({
+  name: 'CommentQuery',
+  fields: {
+    getListOfComments: {
+      description: 'Retrieves comments for given list of ids (default all) in descending order of creation time',
+      type: new GraphQLList(new GraphQLNonNull(CommentType)),
```

Comment:
@Shurtu-gal will this query work, when we have no comments.
What is the reason of adding GraphQLNonNull here?